server/sonar-server/src/main/java/org/sonar/server/user/UserUpdater.java

```diff
@@ -175,45 +178,80 @@ private UserDto createNewUserDto(DbSession dbSession, NewUser newUser) {
     return userDto;
   }
 
-  private void updateUserDto(DbSession dbSession, UpdateUser updateUser, UserDto userDto) {
```

Comment:
check a few lines above it will be in yellow. # reference are not handled properly by GitHub